pages/micromaster/profiles/Jeromephilip.md

```diff
@@ -0,0 +1,46 @@
+# Jerome Philip
+
+## About
+
+_Hello!_ My name is Jerome and I am a high school student at Waterloo Collegiate Institute. Currently, I am finishing my last year and aspiring to
+become a __Software Engineering/Computer Science__ student at the **University of Waterloo**.
```

Comment:
this line break is not required as it is one sentence however will be accepting it for now